### ISSUE TITLE
Don't show lightbox if can't show an image size larger than the current one displayed

### ISSUE THREAD

**Issue by fortes** [Treesaver/treesaver#59]
Looks silly and is confusing.

**Comment by scottkellum** [comment]
good idea, Can you remove the class "lightbox" or give me some handle to remove styles that indicate a photo can be clicked while you are at it.

**Comment by fortes** [comment]
FYI, if you add target="lightbox" to a link, it will be ignored if there's an appropriate lightbox to open (that way you can link to a larger version of the image for legacy browsers): https://github.com/Treesaver/treesaver/wiki/LightBox

Unfortunately, it can be tough to tell whether the lightbox will fit beforehand (i.e. caption could be big) ... so not sure I'll be able to give you a specific class for no zooming...
